seatunnel-connectors/seatunnel-connector-spark-clickhouse/src/main/scala/org/apache/seatunnel/spark/sink/Clickhouse.scala

```diff
@@ -131,12 +198,77 @@ class Clickhouse extends SparkBatchSink {
     schema
   }
 
+  private def getClusterShardList(conn: ClickHouseConnectionImpl, clusterName: String,
+                                  database: String, port: String): ListBuffer[Shard] = {
+    val rs = conn.createStatement().executeQuery(
+      s"select shard_num,shard_weight,replica_num,host_name,host_address," +
+        s"port from system.clusters where cluster = '$clusterName'")
```

Comment:
@BenJFan Is `system.clusters` a fixed value? Is this the only way to get it?

Reply:
Yes. System.clusters is clickhouse's system table. Use for save cluster info which user config